Makefile.js

```diff
@@ -344,9 +344,18 @@ function generatePrerelease(prereleaseId) {
  */
 function publishRelease() {
     ReleaseOps.publishRelease();
+    const releaseInfo = JSON.parse(cat(".eslint-release-info.json"));
+    const isPreRelease = /[a-z]/u.test(releaseInfo.version);
```

Comment:
We can have a more specific check like testing against `/alpha|beta|rc/u` RegExp. But I suppose It will be prerelease anyway if there are alphabets in the version.

Ideally `.eslint-release-info.json` should have a `isPreRelease` property.

Reply:
I think this works. 👍